### PR TITLE
Add faster rcnn voc and coco pretrained models based on resnet50_v1b

### PR DESCRIPTION
**Pull request by ijkguo**
Main changes:
- RPN target generator: match all iou to gt_max, change label assign order
- RCNN target generator: sample valid proposals
- v1b models: replace resnet50_v2a after batch normalization statistics can be fixed from model zoo. (note: when using other models from model zoo as pretrained base, set use_global_stats manually).

### CODE REVIEW

**Comment by mli** [comment]
Job PR-234-2 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-234/2/index.html 
Code coverage of this PR:  vs. Master:
